### ISSUE TITLE
Feature request; Add beta firmwares to "Google images" menu

### ISSUE THREAD

**Issue by s3axel** [badabing2005/PixelFlasher#248]
Would it be possible to add download options for the beta firmwares as well ? Would be extremely handy :)

**Comment by badabing2005** [comment]
It's possible, but unlike the main downloads, the format is not api friendly, the pages need to be scraped, and if anything changes on the page, the scraping would break.
Aside from that, PF caches the data so that it doesn't have to waste time calling out and processing each time.
With beta, the cached data is short lived and subject to Google when it decides to release new version, so unless it's checked each time, which will be slow, the data won't be reliable / useful.

I'll see if something can be done, but no promises, as it provide little value for high cost.
The beta link is already available in the help menu under links, so it's just a few clicks for a user to grab the beta.

**Comment by s3axel** [comment]
Thanks for your reply and understood - much appreciated !

**Comment by VisionR1** [comment]
Hi, with the autopif2 by @osm0sis, can merge it in PixelFlasher and download latest random beta FP from image? Without need to download the whole.

**Comment by badabing2005** [comment]
> Hi, with the autopif2 by @osm0sis, can merge it in PixelFlasher and download latest random beta FP from image? Without need to download the whole.

Yes, and it's already done, it should be released today / tomorrow.

**Comment by aioros57** [comment]
Hello,pixelflasher Jan 28, 2025 v7.9.2.2 change log works well also for other models of smartphone and tablet.
Example for me for my Samsung 10, it detects the system and magisk the modules installed. For the internal files of the root ok also, PIF, scrrpy works too.

**Comment by badabing2005** [comment]
> Hello,pixelflasher Jan 28, 2025 v7.9.2.2 change log works well also for other models of smartphone and tablet. Example for me for my Samsung 10, it detects the system and magisk the modules installed. For the internal files of the root ok also, PIF, scrrpy works too.

Is that a question or a statement?
Yes it detects and works well with other phones.
This ticket is about Pixel Beta firmware download

**Comment by badabing2005** [comment]
Complete in v7.10.0.0
